contracts/interfaces/ILimitOrder.sol

```diff
@@ -81,18 +119,22 @@ interface ILimitOrder is IStrategyBase {
         uint64 expiry;
     }
 
-    /**
-     * Fill limit order by protocol
-     */
+    /// @notice Fill an order by interacting with an external protocol
+    /// @notice Called by user proxy only
+    /// @param _order The order that is going to be filled
+    /// @param _orderMakerSig The signature of the order from maker
+    /// @param _params Protocol specific filling parameters
+    /// @param _crdParams Contains details of the fill permit
     function fillLimitOrderByProtocol(
         LimitOrderLibEIP712.Order calldata _order,
         bytes calldata _orderMakerSig,
         ProtocolParams calldata _params,
         CoordinatorParams calldata _crdParams
     ) external returns (uint256);
 
-    /**
-     * Cancel limit order
-     */
+    /// @notice Cancel an order
+    /// @notice Called by user proxy only
+    /// @param _order The order that is going to be canceled
```

Comment:
```suggestion
    /// @param _order The order that is going to be cancelled
```

Reply:
Seems like the american goes with single L however british goes with double. I'm ok with both.

Reply:
We seemed to be using double in other contracts.

Reply:
fixed ece0b1215ea23b095a27e19155f2732d830a2246